src/Components/SwiperCategories.js

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import { Swiper, SwiperSlide } from "swiper/react";
+import "swiper/css";
+import "swiper/css/pagination";
+import "swiper/css/navigation";
+import { Pagination, Navigation } from "swiper";
+import  categories  from '../mocks/en-us/product-categories.json'
+import { ImgForCategories, StyledH1, StyledH3 } from "../utils/StyledComponents/Styled";
+
+
+export default function App() {
```

Comment:
do you want to be exported as App?

Reply:
Oops, it kept the name from the swiper example, changed.